sources/packages/backend/apps/workers/src/controllers/assessment/assessment.controller.ts

```diff
@@ -495,20 +497,37 @@ export class AssessmentController {
     programYearTotal: ProgramYearTotal,
     output: VerifyAssessmentCalculationOrderJobOutDTO,
   ): void {
-    const PROGRAM_YEAR_TOTAL = "programYearTotal";
     // Create the dynamic variables to be outputted.
-    programYearTotal.awardTotals.forEach((award) => {
+    this.formulateDynamicVariables(programYearTotal.awardTotals, output);
```

Comment:
Just wondering why `formulate` instead of, for instance, `createDynamicVariables` or `generateDynamicVariables`?